programs/svm-spoke/src/instructions/fill.rs

```diff
@@ -193,30 +194,23 @@ fn unwrap_fill_v3_relay_params(
 }
 
 #[derive(Accounts)]
-#[instruction(relay_hash: [u8; 32], relay_data: V3RelayData)]
 pub struct CloseFillPda<'info> {
     #[account(mut, address = fill_status.relayer @ SvmError::NotRelayer)]
     pub signer: Signer<'info>,
 
     #[account(seeds = [b"state", state.seed.to_le_bytes().as_ref()], bump)]
     pub state: Account<'info, State>,
 
-    #[account(
-        mut,
-        seeds = [b"fills", relay_hash.as_ref()],
-        bump,
-        close = signer,
-        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ SvmError::InvalidRelayHash
-    )]
+    #[account(mut, close = signer)]
```

Comment:
it might be a useful to say why this is still safe.

Reply:
I know this is not in the scope of this changes but curious if we could do:
`close = fill_status.relayer` and remove the `address = fill_status.relayer` constraint in the signer

Reply:
anchor `close` method requires `sol_destination` to be `AccountInfo` as it is moving account lamports upon close